### PR TITLE
fix: add missing files for `make bundle`

### PR DESCRIPTION
**Pull request by Madhu-1**
After running `make bundle` locally few files got updated and new files also got added.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

/cc @nbalacha

### CODE REVIEW

**Review by nbalacha — CHANGES_REQUESTED** [review]
Some of these do not need to be updated. They are temporary files generated from templates.
However, [bundle/manifests/lvm-operator.clusterserviceversion.yaml](https://github.com/red-hat-storage/lvm-operator/pull/167/files#diff-f96bdaa8b095a2ecd1bfe9c839663e0352fe0ae96dda277b8de28fd1a7d96ab6) needs to be updated as the image is incorrect.

File: config/default/kustomization.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
```
Comment (nbalacha) [inline config/default/kustomization.yaml:1]: This was updated by the make bundle target as we use different ns for upstream and downstream.

File: bundle/manifests/lvm-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -285,7 +285,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.name
-                image: quay.io/nibalach/lvm-operator:e2e
+                image: quay.io/ocs-dev/lvm-operator:latest
```
Comment (nbalacha) [inline bundle/manifests/lvm-operator.clusterserviceversion.yaml:288]: This file should be committed.

File: config/default/manager_custom_env.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
```
Comment (nbalacha) [inline config/default/manager_custom_env.yaml:1]: This is a file generated by make update-mgr-env and does not need to be committed.

File: config/manager/kustomization.yaml [hunk 4]
```diff
@@ -3,3 +3,10 @@ resources:
```
Comment (nbalacha) [inline config/manager/kustomization.yaml:3]: Not required to be committed.

Comment (Madhu-1) [reply config/manager/kustomization.yaml:3]: @nbalacha how about adding unwanted files to gitignore?

Comment (nbalacha) [reply config/manager/kustomization.yaml:3]: I don't think it is required. If it makes things less confusing, we can merge the temporary files as they will be overwritten anyway by the make targets.

Comment (Madhu-1) [reply config/manager/kustomization.yaml:3]: Yes as you mentioned it's less confusing and helps both contributors and maintainers to keep track of it.

**Review by nbalacha — APPROVED** [review]

**Pull request merged**